config/initializers/clamscan.rb

```diff
@@ -2,4 +2,5 @@
 
 ClamScan.configure do |clam_config|
   clam_config.client_location = Settings.binaries.clamdscan
+  clam_config.default_scan_options = { stdout: false }
```

Comment:
I'm unsure if this is really necessary. But stdout pollution is evil. So I suppressed it.